tests/fv3_conf/cpld_bmark_tiled_run.IN

```diff
@@ -0,0 +1,101 @@
+mkdir INPUT RESTART history MOM6_OUTPUT
```

Comment:
We should come up with an easier way to set up those model runs, this is fairly complex.

Reply:
All the coupled tests use this sort of set-up. Each component has its preferred name/location for input, output and restarts. I do agree it would be nice if we could use a more standardized approach. For example, right now some FV3 input is expected in INPUT, some is expected in the run directory itself. CICE6 and CMEPS use pointer files to determine the location of the RESTART files. It's all a muddle.